ansible/roles/install_poetry/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+- name: Check cloned git
+  become: true
+  stat:
+    path: /home/{{ user_name }}/git
+  register: repo_result
+- name: Clone git repo
+  become: true
+  git:
+    repo: "{{ git_clone_url }}"
+    dest: /home/authz/git
+    version: master
```

Comment:
Lets introduce a variable here with which we can set code for what branch we will deploy

Reply:
introduced